### PR TITLE
Offset: Eliminate little-used internal function

### PR DESCRIPTION
**Pull request by gibson042**
Fixes gh-3449

### CODE REVIEW

File: src/offset.js [hunk 1]
```diff
@@ -183,7 +176,9 @@ jQuery.each( { scrollLeft: "pageXOffset", scrollTop: "pageYOffset" }, function(
 
 	jQuery.fn[ method ] = function( val ) {
 		return access( this, function( elem, method, val ) {
-			var win = getWindow( elem );
+
+			// Coalesce documents and windows
+			var win = jQuery.isWindow( elem ) ? elem : elem.nodeType === 9 && elem.defaultView;
```
Comment (markelog) [inline src/offset.js]: Maybe `if..else` instead of ternary... please :innocent:

Comment (gibson042) [inline src/offset.js]: Done.

File: src/offset.js [hunk 2]
```diff
@@ -178,7 +178,12 @@ jQuery.each( { scrollLeft: "pageXOffset", scrollTop: "pageYOffset" }, function(
 		return access( this, function( elem, method, val ) {
 
 			// Coalesce documents and windows
-			var win = jQuery.isWindow( elem ) ? elem : elem.nodeType === 9 && elem.defaultView;
+			var win;
+			if ( jQuery.isWindow( elem ) ) {
+				win = elem;
+			} else {
+				win = elem.nodeType === 9 && elem.defaultView;
```
Comment (markelog) [inline src/offset.js]: Yeysies

Comment (mgol) [inline src/offset.js]: Maybe the nodeType check could be moved into the else? That would make it even less hacky.

Comment (gibson042) [inline src/offset.js]: D'oh, good call.

**Pull request merged**